src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionIT.java

```diff
@@ -355,6 +412,60 @@ void assignAfterSubscriptionTest(VertxTestContext context) throws InterruptedExc
         context.completeNow();
     }
 
+    @Test
+    void assignEmptyAfterSubscriptionTest(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String topic = "subscribe-and-assign-empty-topic";
+
+        KafkaFuture<Void> future = adminClientFacade.createTopic(topic, 4, 1);
+        future.get();
+
+        String consumerName = "my-kafka-consumer-assign-empty";
+
+        JsonObject consumerJson = new JsonObject();
+        consumerJson.put("name", consumerName);
+        consumerJson.put("format", "json");
+
+        JsonObject partitionsRoot = new JsonObject();
+        JsonArray partitions = new JsonArray();
+
+        partitionsRoot.put("partitions", partitions);
```

Comment:
compared to the previous test, there is no assignment to a partition and then assignment with empty list to validate the feature.

Reply:
just added the section, thank you!